bedrock/security/templates/security/bug-bounty/web-eligible-sites.html

```diff
@@ -166,11 +166,31 @@ <h3>Payment Subscription</h3>
     <li>subscriptions.firefox.com</li>
   </ul>
 
-  <h3>Pocket</h3>
-  <ul class="mzp-u-list-styled">
-    <li>getpocket.com</li>
-    <li>api.getpocket.com</li>
-    <li>widgets.getpocket.com</li>
+  <h3 id="pocket"><b>Pocket</b></h3>
```

Comment:
If there's a partic need for Pocket to have a different emphasis from the other `h3`s, we should not bold it. If it does need it, we can support it via a CSS style

```suggestion
  <h3 id="pocket">Pocket</h3>
```

Reply:
Visually, with and without `<b>` are identical to me (Nightly, MacOS)

Reply:
for me too, I was planning to have Pocket in its own section then we needed to change that and I forgot to remove the b tag